### ISSUE TITLE
Narrator does not read the error message displayed while we enter incorrect email address.

### ISSUE THREAD

**Issue by bhagwatshelke1047** [OfficeDev/office-ui-fabric-react#10702]
Environment Details: 
Application Name: Customer Service Insights
Windows Version: Windows10
Browser Versions: Microsoft Edge 44.18362.267.0
Screen Reader Version: Narrator
Mas violated : MAS 3.3.1

Repro Steps:
1. Open application Customer Service Insights.​
2. Home page should open for Customer Service Insights.​
3. Navigate to Global Navigation Bar at the top of the screen and select "Workspaces Panel".​
4. Delete any workspace by hitting "Share" button on any workspace. "Share this workspace" popup screen should appear.​
5. Select "Share" tab item.​
6. Verify that all the elements of "Share Workspace-Share" are accessible and MAS Compliant.

Actual:
 Narrator does not read the error message displayed while we enter incorrect email address.

Expected:
Narrator should read the error message displayed while we enter the incorrect email address.

User Impact:
The screen reader users will not understand the entered email is wrong as narrator does not announce the error message.

**Comment by khmakoto** [comment]
Hi @bhagwatshelke1047, I think what you're logging is an application bug that needs to be investigated by someone from the Customer Service Insights team since we it is not clear if this is indeed a Fabric bug. Please contact that team first and, if it is indeed a Fabric issue then we can start working on it. Thanks!

**Comment by bhagwatshelke1047** [comment]
+@Ge Wu<mailto:gew@microsoft.com>

Hi Ge,

Can you please look into this https://github.com/OfficeDev/office-ui-fabric-react/issues/10702 and provide the necessary details.

Regards,
Bhagwat Shelke
TCS - Microsoft Global Relationship | v-bhshel<mailto:v-bhshel>
C+AI Accessibility

[cid:image001.jpg@01D57AAD.3D4EC140]

**Comment by easy0924** [comment]
Hi, this is related to another bug here, and I've left a comment
https://github.com/OfficeDev/office-ui-fabric-react/issues/10615#issuecomment-538082998

So the issue is related is because when no result dropdown shows up, the aria-controls and aria-owns value and invalid, which causes screen reader to not associate the no result dropdown with the input, thus no announcement of the no result message

![image](https://user-images.githubusercontent.com/2107890/66156192-0b7cc880-e5d6-11e9-81a8-1e636cbb05de.png)

![image](https://user-images.githubusercontent.com/2107890/66156268-3404c280-e5d6-11e9-8196-c429b80ddf8c.png)

**Comment by bhagwatshelke1047** [comment]
Yeah, commented.

Thanks
Ge

From: Bhagwat Shelke (Tata Consultancy Services Ltd) <v-bhshel@microsoft.com>
Sent: Thursday, October 3, 2019 11:51 PM
To: OfficeDev/office-ui-fabric-react <reply@reply.github.com>; OfficeDev/office-ui-fabric-react <office-ui-fabric-react@noreply.github.com>; Ge Wu <gew@microsoft.com>
Cc: Mention <mention@noreply.github.com>
Subject: RE: [OfficeDev/office-ui-fabric-react] Narrator does not read the error message displayed while we enter incorrect email address. (#10702)

+@Ge Wu<mailto:gew@microsoft.com>

Hi Ge,

Can you please look into this https://github.com/OfficeDev/office-ui-fabric-react/issues/10702<https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FOfficeDev%2Foffice-ui-fabric-react%2Fissues%2F10702&data=02%7C01%7Cgew%40microsoft.com%7C0cb0ae1bcf0b4e54e85308d748972f38%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637057686486256829&sdata=7jTRGWI%2Fq%2FuqmrV6ihl8sVVxso4X7tLTqhVDbPIqmmM%3D&reserved=0> and provide the necessary details.

Regards,
Bhagwat Shelke
TCS - Microsoft Global Relationship | v-bhshel<mailto:v-bhshel>
C+AI Accessibility

[cid:image001.jpg@01D57AA1.52AE5DE0]

**Comment by khmakoto** [comment]
Hi @bhagwatshelke1047 @easy0924 I've confirmed the issue reproduces in our website. I'll have a fix shortly.

**Comment by khmakoto** [comment]
I spoke offline with @joschect who's working on the fix for #10615. Since it seems like the fix will solve both problems he'll link this issue as well to that PR.
